src/gamemode.c

```diff
@@ -562,5 +604,11 @@ game_mode_create (GDBusConnection *connection)
   gamemode = g_object_new (game_mode_get_type (), NULL);
   gamemode->client = client;
 
+  g_signal_connect (client, "g-properties-changed",
+                    G_CALLBACK (client_properties_changed), NULL);
+
+  // Should not normally happen but just in case, check for active games on startup
```

Comment:
I don't think this comment is particularly necessary, please remove it

Reply:
Done :+1: